help/md/fnFreeBinary.md

```diff
@@ -1,9 +1,42 @@
 ## name
 fnFreeBinary
+
 ## title
+Free Binary transition rate matrix
+
 ## description
+Constructs a transition rate matrix between two states.
+
 ## details
+ This function creates a transition rate matrix (between two states) for implementation in modeling evolutionary processes. This function accepts non-normalized transition rates which can then be normalized. This normalization process is a key component of ensuring that the total rates (rows) sum to 0 (requirement for transition rate matrices).
+ 
+ It takes in two arguments: 
+    (1) transtion_rates (tr) - A real number that represents the rate of transition between states.
+    (2) rescaled - A boolean vlue that indicates whether or not the matrix should be normalized. Takes on TRUE by default.
+
+0 and 1 represent our 2 states:
+Q = [[-q_{01}, q_{01}],[q_{10}, -q_{10}]]
+
 ## authors
+Sigournie Brock
```

Comment:
I think the convention we are using is to list the authors of the code, rather the authors of the help file. Often, it's not clear who wrote the code, in which case it's okay to leave the entry empty.